### DISCUSSION TITLE
[Enhancement] [tree-select] collapse-tags-tooltip弹出的信息框无法限制大小

### DISCUSSION THREAD

**Discussion by 2292777095** [element-plus/element-plus#13813] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `tree-select`

## Description

目前设置collapse-tags-tooltip的组件，弹出的信息框都没有大小限制，很难看，而且popper相关的属性设置后无效！！！

### Additional comments

_(empty)_

**Comment by chenxch** [comment]
或许可以给collapse-tags-tooltip添加一个class属性方便处理该场景。

**Comment by KeroZhai** [comment]
如果有内置的 `ElScrollbar` 就更好了
